activities/twenty_three_and_me/signals.py

```diff
@@ -0,0 +1,21 @@
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from data_import.signal_helpers import task_signal
+
+from .models import UserData, DataFile
+
+
+@receiver(post_save, sender=UserData)
+def post_save_cb(instance, created, raw, **kwargs):
+    """
+    Create data retrieval task when 23andMe UserData's data is updated.
+    """
+    if not instance.file_url:
```

Comment:
Maybe do this pre_save and check if it's changed, as we've done for the new American Gut UserData?